<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultWindow</class>
 <widget class="QWidget" name="ResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="resultpic" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#resultpic{background-image: url(:/images/resultwindow/result_background.png);}</string>
   </property>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>返回菜单</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:50px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>游戏结束</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>520</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>上传成绩</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt1">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>420</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="cnt5">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_title">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>分数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:Red;
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
